### ISSUE TITLE
Unable to import on UB 22 & Py 3.10

### ISSUE THREAD

**Issue by olgabaranov** [Ad115/EnsemblRest#3]
When importing on Python 3.10.12, fails with following message: 
```
>>> import ensembl_rest
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/xxx/.local/lib/python3.10/site-packages/ensembl_rest/__init__.py", line 1, in <module>
    from .ensemblclient import EnsemblClient, _api_table
  File "/home/xxx/.local/lib/python3.10/site-packages/ensembl_rest/ensemblclient.py", line 11, in <module>
    from importlib_resources import open_text, path
ImportError: cannot import name 'open_text' from 'importlib_resources' (/home/xxx/.local/lib/python3.10/site-packages/importlib_resources/__init__.py)
```

can be fixed by changing `importlib_resources` to `importlib.resources` in line 11 of /home/xxx/.local/lib/python3.10/site-packages/ensembl_rest/ensemblclient.py

versions I am using (fresh pip install on Ubuntu 22.04):
importlib-resources     6.1.0
ensembl-rest            0.3.3

**Comment by Ad115** [comment]
Related to #4 , PR #5  addresses this issue

**Comment by Ad115** [comment]
Merged PR #5. Will update the pip package shortly
